File: CRUD/Java usando NetBeans/Aplicacion Web usando MVC Spring V2/SistemaWebSpring/src/main/java/sistema/models/TipoUsuario.java
// Written by Ismael Heredia in the year 2017
package sistema.models;

public class TipoUsuario {
  
    int id;
    String nombre;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
    
    public TipoUsuario() {
        
    }

    public TipoUsuario(int id, String nombre) {
        this.id = id;
        this.nombre = nombre;
    }

    @Override
    public String toString() {
        return "TipoUsuario{" + "id=" + id + ", nombre=" + nombre + '}';
    }
  
}

File: CRUD/PHP/Ajax V3/entities/TipoUsuario.php
<?php

class TipoUsuario {

  private $id;
  private $nombre;

  public function __construct(){
    $this->id = "";
		$this->nombre = "";
   }

  public static function CreateTipoUsuario($id,$nombre) {
   		$instance = new self();
    	$instance->id = $id;
		  $instance->nombre = $nombre;
    	return $instance;
  }

  public function getId() {
    	return $this->id;
  }

  public function setId($id) {
    	$this->id = $id;
  }
  
  public function getNombre() {
   	 	return $this->nombre;
  }

  public function setNombre($nombre) {
    	$this->nombre = $nombre;
  }
    
  public function __destruct(){
    $this->id = "";
		$this->nombre = "";
   }  
   
   public function toString() {
		return "TipoUsuario{" . "id=" . $this->id . ", nombre=" . $this->nombre . '}';
   }
  
}

?>